app/Filament/Components/Actions/ExportEggAction.php

```diff
@@ -20,6 +21,11 @@ protected function setUp(): void
         parent::setUp();
 
         $this->label(trans('filament-actions::export.modal.actions.export.label'));
+        $this->iconButton();
+
+        $this->icon('tabler-file-export');
+
+        $this->iconSize(IconSize::ExtraLarge);
```

Comment:
The export action is used as a record action and header action. Because of this the icon is too big when used as record action.

<img width="366" height="365" alt="Image" src="https://github.com/user-attachments/assets/67a80088-adc4-48fd-badf-dafae4e608c1" />